packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -115,9 +116,32 @@ router.get(
 router.post(
   "/patient-discovery/response",
   asyncHandler(async (req: Request, res: Response) => {
-    const patientDiscovery = patientDiscoveryResponseSchema.parse(req.body);
+    console.log("Received patient discovery response from IHE Gateway", req.body);
+    const pdResponse = patientDiscoveryResponseSchema.parse(req.body);
+    await createPatientDiscoveryResult(pdResponse);
+
+    return res.sendStatus(httpStatus.OK);
+  })
+);
+
+// router.post(
+//   "/patient-discovery/response-trigger",
+//   asyncHandler(async (req: Request, res: Response) => {
+//     const organization = await getOrganizationOrFail({ cxId });
+//     const patient = await getPatientOrFail({ id: patientId, cxId });
+
+//     await cqCommands.patient
+//       .discover(patient, organization, facilityNpi)
+//       .catch(processAsyncError(`cq.patient.discover`));
+//     return res.sendStatus(httpStatus.OK);
+//   })
+// );
 
-    await handlePatientDiscoveryResponse(patientDiscovery);
+router.post(
+  "/patient-discovery/response-trigger",
+  asyncHandler(async (req: Request, res: Response) => {
+    const iheGateway = makeIheGatewayAPI();
+    iheGateway.startPatientDiscovery();
```

Comment:
This is odd or I might be missing something:
1. why `startPatientDiscovery` here if it's triggered upon Patient create?
2. if we're to start a PD here, wouldn't it make sense to have just `POST /patient-discovery`?